404.html

```diff
@@ -5,9 +5,9 @@
 ---
 <div class="row">
   <div class="small-12 columns">
-     <h1>We could not find that page</h1>
-     <p>
+     <h1 style="text-align: center;">We could not find that page</h1>
+     <article style="text-align: center;">
```

Comment:
All our other pages use left-aligned text so I'd like to keep it that way here too, can you please remove that part?

I'm fine with the p -> article change.

Reply:
yeah sure I'll make the changes back to left align